src/applications/personalization/dashboard/sass/dashboard.scss

```diff
@@ -4,6 +4,7 @@
 
 #dashboard-title {
   margin-top: 1.5em;
+  outline: 0;
```

Comment:
Taking the advice from [this article](https://css-tricks.com/focus-styles-non-interactive-elements/#article-header-id-0) and removing the focus outline since this isn't a clickable element